pyiron/vasp/base.py

```diff
@@ -1068,6 +1084,28 @@ def _set_kpoints(
                         val=" ".join([str(kpt[0]), str(kpt[1]), str(kpt[2]), str(wt)]),
                     )
 
+    def _get_trace_for_kpoints(self, trace):
+        """
+        gets the trace for k-points line mode in a VASP readable form.
+
+        Args:
+            trace (list): ordered list of names for k-points trace
+
+        Returns:
+            list: trace points coordinates for VASP
+        """
+        for t in trace:
+            if not t in self.structure.get_high_symmetry_points().keys():
```

Comment:
pythonic ways is to write  `if t not in self.structure.get_high_symmetry_points().keys()`

Reply:
done